pkg/kube/client.go

```diff
@@ -451,28 +449,6 @@ func (c *client) PodLogs(ctx context.Context, podName, podNamespace, container s
 	return builder.String(), nil
 }
 
```

Comment:
could we retain this function (as a wrapper) and make it public? i need the function for bug-report tool and it's probably generally useful.

Reply:
@ostromart ProxyGet is now natively available so you can directly use it. eg:
```
c.CoreV1().Pods(pod.Namespace).ProxyGet("", pod.Name, "15014", "/version", nil).DoRaw(ctx)

c.CoreV1().Pods(istiod.Namespace).ProxyGet("", istiod.Name, "15014", path, nil).DoRaw(ctx)
```